rc/extra/racer.kak

```diff
@@ -127,20 +125,16 @@ define-command racer-show-doc -docstring "Show the documentation about the rust
     }
     %sh{
         dir=${kak_opt_racer_tmp_dir}
-        (
-            printf -v NEWLINE '\n'
-            cursor="${kak_cursor_line} ${kak_cursor_column}"
-            racer_data=$(racer --interface tab-text  complete-with-snippet  ${cursor} ${kak_buffile} ${dir}/buf | head -n 2 | tail -n 1)
-            racer_match=$(printf %s\\n "$racer_data" | cut -f1)
-            printf %s\\n "info %@.$racer_match.@"
-            if [[ "$racer_match" == "MATCH" ]]; then
-              racer_doc=$(printf %s\\n "$racer_data" | cut -f9 | xargs )
-              racer_doc=$(printf %s\\n "$racer_doc" | sed -e "s/\\\n/\\$NEWLINE/g")
-              racer_doc=$(printf %s\\n "$racer_doc" | sed -e "s/@/\\@/g")
-              printf %s\\n "info %@$racer_doc@"
-            else
-              printf %s\\n "echo -debug 'racer could not find a definition'"
-            fi
-        )
+        cursor="${kak_cursor_line} ${kak_cursor_column}"
+        racer_data=$(racer --interface tab-text  complete-with-snippet  ${cursor} "${kak_buffile}" "${dir}/buf" | head -n 2 | tail -n 1)
```

Comment:
It's not critical, but the `head -n 2 | tail -n 1` can be replaced with `sed -n 2p`, assuming the goal is to print the second line.

Reply:
Changed. (Yes, only the second line is need; the first line has headers and the rest are not the proper matches)